src/panels/lovelace/cards/hui-entities-card.ts

```diff
@@ -234,7 +234,7 @@ class HuiEntitiesCard extends LitElement implements LovelaceCard {
         display: flex;
         flex-direction: column;
         justify-content: space-between;
-        overflow: hidden;
+        overflow-y: visible;
```

Comment:
Why not just `overflow-x: hidden`? 
You now also removed the overflow of the x-axis, and the current rule is the default, so does not do anything?

Reply:
With `overflow-x: hidden` the dropdown box still gets clipped. Actually removing `overflow: hidden` solves the main issue. With default behavior I mean `overflow: hidden` currently actually gets ignored until the elements get re-rendered. So either removing `overflow: hidden` or setting `overflow-y: visible` (not really needed, but makes it implicit) solves the issue I described. However, it does not break or change the current 'default' behavior.

Reply:
It does change the default behavior. We should either change it to `overflow-x: hidden;` or leave it as is.

Reply:
The main issue is that dropdown box get clipped when the card is re-rendered. This is not default behavior. `overflow:hidden` causes this. Changing this to `overflow-x: hidden` will still cause this. Solution is to remove the `overflow: hidden`.

Reply:
The solution is to no re-rerender the card. We are not going to change this just for custom components. If you can show me an example with plain Home Assistant we can discuss this.

Reply:
Re-rendering seems to be the only solution that is available to apply custom `css` to cards. For example this HACS front-end extension has such functionality: https://github.com/Kibibit/kb-frosted-cards. When this happens, the issue with `overflow: hidden` becomes noticeable. Removing `overflow: hidden` fixes this. It basically means, `overflow: hidden` is wrong to begin with. There is no other impact, it only fixes issues when cards get re-rendered, so what prevents you from accepting this PR?

Reply:
This is the reason for that code, and why you can't just remove it:
![image](https://user-images.githubusercontent.com/5662298/111526194-b9dee400-875e-11eb-916c-a5bdd43fae55.png)

Reply:
That is a separate issue. With `overflow: hidden` that text would also be clipped. Therefore the text becomes unreadable when its hidden, so it is unusable to the user. However that is not an issue with `ha-card`. That is an issue with the inner element. In this case most likely `<div class="target">`, it contains `:host` which applies `white-space: nowrap;`. That needs to be removed. Then your text becomes readable again. Or you need to make your screen wider.

Reply:
Related: <https://github.com/home-assistant/frontend/pull/8369>